backend/src/routes/auth_routes.py

```
@@ -0,0 +1,85 @@
+
+    db.add(new_user)
+    db.commit()
+    db.refresh(new_user)
+    print("New user Created")
+	# Re-fetches the object from the DB and updates it in memory varna shayd id none reh jaye jasie apan nodejs me vapas fetch karte the taki updated user mile
+    return new_user # only those fields allowed by pydantic model will be returned
+
+
+#NOTE:OAuth2PasswordRequestForm internally needs username and password only to agar email bhi bhejna hai under username hi bhejna padega in frontend
+@authRouter.post("/login",response_model=schema.Token)
+def login(form_data:OAuth2PasswordRequestForm=Depends(),db:Session=Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)